src/bokeh/core/property_aliases.py

```diff
@@ -50,13 +51,22 @@
 
 Pixels = NonNegative(Int)
 
+
+HAnchor = Either(Enum(enums.Align), Enum(enums.HAlign), Percent)
+VAnchor = Either(Enum(enums.Align), Enum(enums.VAlign), Percent)
```

Comment:
I guess practically it does not really matter, but it does seem odd to have an `Either` with two enums that overlap (both align enums have "center" unless I am mistaken)

Reply:
This is modeled after CSS and was introduced with the new layout. It's not that odd if `Align` can be used on its own, where `{H,V}Align` don't apply. Otherwise it provides `start`/`left`, `start`/`top` and `end`/`right`, `end`/`bottom` aliases.